hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/MethodBase.java

```diff
@@ -84,9 +84,9 @@ abstract class MethodBase implements ServerCalls.UnaryMethod<ByteBuffer, ByteBuf
      * @param methodName a non-null reference to the method name
      */
     MethodBase(
-            @Nonnull final String serviceName,
```

Comment:
I really hate it, when complex PRs that change fundamental functionality are bloated with these kind of clean-up changes. We should coordinate our changes better.